### ISSUE TITLE
Could we move the letter to an open platform or at least add a link for non-tech/non-github users?

### ISSUE THREAD

**Issue by albertbuchard** [nadimkobeissi/appleprivacyletter#5623]
Hi,

There is only 6K signatures, and I believe this number could be greatly increased by simplifying the signing process for non-github and non-tech savy users. Maybe using https://www.change.org/ ?

**Comment by peteeckel** [comment]
I absolutely support that idea. The issue is too important for artificially limiting the potential signers to GitHub users and those willing to sign up to GH for the sole purpose of signing the petition. We need to exercise as much pressure on Apple as possible.

**Comment by nadimkobeissi** [comment]
Will address this tonight

**Comment by Jinxiansen** [comment]
Yes, I agree.

**Comment by dwitt1915** [comment]
I agree also! It is VERY important that we get on board to stop Apple from building a back door to all the world's iPhones!

**Comment by annaproxy** [comment]
Any updates on this?

**Comment by xiaoyu2006** [comment]
I think the easiest way is to set up a kind of third party auth (Twitter, etc.) Users (non-programmers) can OAuth into the system and leave their signature.

However it requires a backend.

**Comment by xiaoyu2006** [comment]
Another solution I come up with is to create a new org and people post issues using the org’s API key. We also need a captcha.

However it sounds insecure and is easy to be abused.

**Comment by peteeckel** [comment]
Those ideas may all work, but they are IMHO too much of a hurdle and will deter most non-technical people from sigining. What we need is a low-threshold solution that everyone can use with at most a handful of mouse-clicks. Although certainly not the most elegant solution, spawning a petition e.g. on change.org and linking here would probably do the trick.

We need signers, and we need them quickly. I'm afraid the initial momentum is already running out.

**Comment by xiaoyu2006** [comment]
Yes time is limited. I think change.org is fine enough.

**Comment by xiaoyu2006** [comment]
I have done some copy-and-paste work at http://chng.it/5jFtRddPkG. (Removed some quotes.) If you have a better formatted one I will unpublish this.

**Comment by TeleTillYerSmelly** [comment]
> I absolutely support that idea. The issue is too important for artificially limiting the potential signers to GitHub users and those willing to sign up to GH for the sole purpose of signing the petition. We need to exercise as much pressure on Apple as possible.
From what I've seen, we're toast.  The media outlets are ignoring us.  I could find very few articles and almost ZERO responses to those articles on Linked-In (much/most of our Management & many peers live there).  Sure when I searched specifically on other search engines, I could find those who support us posting, articles, etc., but I did a search on StartPage for the Screeching voice of the minority and the second page provided a bunch of porn links.  (hate to say it, but I was depressed from my recent searches & research)
No, we should not give up, but just face this is an uphill battle.  No more suggesting Apple products to our customers, move to Linux supported hardware and software...what else can we do?

**Comment by Tyil** [comment]
Would anyone object to me setting up a mailing list, and accepting signatures through that? This would open up a free (as in freedom) way for people to sign, without having to sign up to any particular service.

**Comment by xiaoyu2006** [comment]
That sounds nice.

Yi, sent from iPhone.

**Comment by TeleTillYerSmelly** [comment]
For MSM, IMO an excellent article:  https://www.washingtonpost.com/opinions/2021/08/19/apple-csam-abuse-encryption-security-privacy-dangerous/
And hats off to AsuharietYgvar AppleNeuralHash2ONNX and those who assisted on the hash collision work.
Perhaps the sky is not falliing yet?

**Comment by Tyil** [comment]
I've set up [a mailing list through Sourcehut](https://lists.sr.ht/~tyil/apple-privacy-letter), to allow people to sign without having to sign up to a github account. If we document the existence of this mailing list properly, it can also be shared with less technical people, so they too can show their support.

**Comment by xiaoyu2006** [comment]
@Tyil Nice. Hope this can be adopted by @nadimkobeissi

**Comment by annaproxy** [comment]
EFF also has a petition up here:
https://act.eff.org/action/tell-apple-don-t-scan-our-phones
